### DISCUSSION TITLE
Is there a limit in the number requests to the API?

### DISCUSSION THREAD

**Discussion by RS-SY92** [openpotato/openplzapi.website#6] [Q&A] [answered]
Hi!
first of all thank you for your efforts to provide this open data service. We would like to use the API in a project where we have to validate ZIP code and city name data for specific data sets. In our context we are expecting the we need to validate several hundred data sets per day. Would this be an issue to to a limitation of API-requests that can be send per time interval?
Thank you in advance for your response.

**Comment by fstueber** [comment, marked as answer]
Hi, thanks for your interest in this project. Rate limit at the moment is 5 requests per second. Does this fit to your needs?

**Reply by RS-SY92** [reply]
Hi and thanks for the quick reply. That is definitely sufficient for our current needs! Thank you again!
